census_view.cpp

```diff
@@ -1075,8 +1075,24 @@ Input* CensusView::class_parms_from_class_name(std::string const& class_name)
 bool CensusView::column_value_varies_across_cells(std::string const& header) const
 {
     auto const z = case_parms()[0][header];
-    for(auto const& j : class_parms()) {if(z != j[header]) return true;}
-    for(auto const& j : cell_parms() ) {if(z != j[header]) return true;}
+
+    size_t const class_header_index = class_parms().begin()->get_member_index(header);
```

Comment:
Can we be sure that `class_parms()` here (and `cell_parms()` below) are not empty?

Reply:
Looks like we can:
```cpp
multiple_cell_document::multiple_cell_document()
    :case_parms_  (1)
    ,class_parms_ (1)
    ,cell_parms_  (1)
{
    assert_vector_sizes_are_sane();
}
...
multiple_cell_document::multiple_cell_document(std::string const& filename)
{
    xml_lmi::dom_parser parser(filename);
    parse(parser);
}
...
void multiple_cell_document::parse(xml_lmi::dom_parser const& parser)
{
...
    assert_vector_sizes_are_sane();
}
...
void multiple_cell_document::assert_vector_sizes_are_sane() const
{
    LMI_ASSERT(1 == case_parms_.size());
    LMI_ASSERT(    !class_parms_.empty());
    LMI_ASSERT(    !cell_parms_ .empty());
}
```